### PR TITLE
feat(zuora):support zuora

### PR DESCRIPTION
**Pull request by ArielCabib**

### CODE REVIEW

File: config/job_config_sample.yaml [hunk 1]
```diff
@@ -59,6 +59,14 @@ inputs:
       keySpace: keySpace
       options:
         ....
+  input_7:
```
Comment (RonBarabash) [inline config/job_config_sample.yaml]: @ArielCabib can u add also the README docs?

File: src/main/scala/com/yotpo/metorikku/configuration/job/input/Zuora.scala [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.yotpo.metorikku.configuration.job.input
+
+import com.yotpo.metorikku.configuration.job.InputConfig
+import com.yotpo.metorikku.input.Reader
+import com.yotpo.metorikku.input.readers.zuora.ZuoraInput
+
+case class Zuora(email: String,
```
Comment (lyogev) [inline src/main/scala/com/yotpo/metorikku/configuration/job/input/Zuora.scala]: Can you move all of them to be Option[String]?

File: build.sbt [hunk 3]
```diff
@@ -37,6 +37,7 @@ libraryDependencies ++= Seq(
   "org.apache.spark" %% "spark-sql-kafka-0-10" % sparkVersion % "provided" excludeAll(excludeJpountz),
   "org.apache.spark" %% "spark-streaming" % sparkVersion % "provided",
   "org.apache.spark" %% "spark-avro" % sparkVersion % "provided",
+  "com.springml" % "spark-zuora_2.11" % "1.1.0",
```
Comment (lyogev) [inline build.sbt]: Wondering if this should be provided...

**Comment by lyogev** [comment]
You need to remove HTML files from assembly

File: build.sbt (hunk 3 above)
Comment (RonBarabash) [reply build.sbt]: @lyogev if it has different versions for different spark versions than yes

**Comment by ArielCabib** [comment]
@lyogev what HTML? I didn't commit any HTML

**Pull request closed without merging**